TECHNICAL-WRITING-GUIDE.md

```diff
@@ -0,0 +1,155 @@
+should:
+
+- be written for all experience levels
+- be written in first person plural (e.g "we")
+- avoid placing "blame" on the user
+- avoid excessive jokes and informal jargon
+
+
+**2. Is it helpful?**
+
+Will this document or update help users to understand a new concept, set up
+something from scratch, or answer a question?
+
+Writers and editors for Kong docs should strive to cover their topic throughly 
+and succinctly. The goal is to help users accomplish a task from start to finish
+or learn something rather than just copy and pasting bits of code. Documentation
+should also help guide users to other relevant documentation or resources.
+
+**3 - Is it true?**
+
+Kong documentation should be technically correct and contain enough detail to 
+provide users with the correct context. Before publishing, writers should have
+other contributors review and verify the technical accuracy of a document. This
+ensure that users will be able to follow a doc from start to finish.
+
+## Types of Documentation and Structure
+Kong docs contain four “types” of documentation
+
+
+### Reference documentation
+
+This type of documentation is strictly informative such as documenting API
+endpoints or configuration properties.
+
+
+### Explanatory documentation
+
+Explanatory guides provide in-depth clarification and examples, generally split
+up into multiple sections.
+
+Explanatory documentation should include:
+
+* Title
+* Section 1
+    * Elaboration
+    * Example
+* Section ...
+    * Elaboration
+    * Example
+* Section N
+    * Elaboration
+    * Example
+* Related article
+
+Avoid overly specific edge cases as generally these are more appropriate for
+How-To guides or Tutorials. Each section should begin with a topic sentence
+followed by one or more paragraphs that elaborate on the topic and provide
+examples. The purpose of explanatory documentation is to help a user understand
+a specific concept.
+
+
+### How-to Guides
+
+How-to guides should solve an explicit problem or accomplish a specific task
+such as “How to Delete an Admin”
+They should follow the follow structure:
+
+* Title
+* Table of Contents
+* Introduction
+* Prerequisites
+* Step 1 
+* Step 2
+* Step ...
+* Step n
+* Optional next steps or related article
+
+Our _How-To template_ have this layout available in Markdown for you to use as
+a starting point.
+
+*Title [h1]*
+
+How-To guide titles typically follow the “How to [X] using [Y]” format.
+
+*Table of Contents [h4]*
+ 
+ - Bulleted list of each step in the guide
+
+*Introduction [h3]*
+
+How-To guides should start with an introduction. It should be limited to a few
+succinct sentences or paragraphs that introduces what the user will be learning
+to do. A “how-to” guide should provide an end-goal, or something that the user
+will accomplish by following the steps. 
+
+*Prerequisites [h3]*
+
+* A bulleted list of everything a user needs to complete the how-to guide
```

Comment:
We should mention the possible subsections addressed in https://github.com/Kong/docs.konghq.com/pull/1224#discussion_r273739963

```suggestion
* A bulleted list of everything a user needs to complete the How-To guide.
```